ts/Series/SolidGauge/SolidGaugeSeriesDefaults.ts

```diff
@@ -0,0 +1,224 @@
+/* *
+ *
+ *  Solid angular gauge module
+ *
+ *  (c) 2010-2021 Torstein Honsi
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type SolidGaugeSeriesOptions from './SolidGaugeSeriesOptions';
+
+/* *
+ *
+ *  API Options
+ *
+ * */
+
+/**
+ * A solid gauge is a circular gauge where the value is indicated by a filled
+ * arc, and the color of the arc may variate with the value.
+ *
+ * @sample highcharts/demo/gauge-solid/
+ *         Solid gauges
+ *
+ * @extends      plotOptions.gauge
+ * @excluding    dial, pivot, wrap
+ * @product      highcharts
+ * @requires     modules/solid-gauge
+ * @optionparent plotOptions.solidgauge
+ */
+const SolidGaugeSeriesDefaults: SolidGaugeSeriesOptions = {
+    /**
+     * The inner radius for points in a solid gauge. Can be given as a
+     * percentage string like `"100%"`.
+     *
+     * @sample {highcharts} highcharts/plotoptions/solidgauge-radius/
+     *         Individual radius and innerRadius
+     *
+     * @type      {string}
+     * @default   60
```

Comment:
Shouldn't it be `"60%"` instead of `60`?

Reply:
Good question. I will check, maybe it supports indeed numbers, but only as percentage.

Reply:
It does actually support numbers as percentages. Which I think is a flaw, because elsewhere in similar Highcharts settings, numbers are interpreted as pixels, and percentages are strings. Not sure to do about this. Maybe we should just remove number from the type definition and description text and leave the functionality as is?

Reply:
I agree. Should I change the defaults to strings?

Reply:
Yes I think we should